pkg/pool-weighted/contracts/lbp/LBPool.sol

```diff
@@ -85,12 +85,22 @@ contract LBPool is WeightedPool, Ownable2Step, BaseHooks {
     /// @dev Indicates that the `owner` has disabled swaps.
     error SwapsDisabled();
 
+    /// @dev Indicates that removing liquidity is not allowed.
+    error RemovingLiquidityNotAllowed();
```

Comment:
Also as a style not, type definitions go on top, then state variables, then events, then errors, then functions.